importScripts/sampleRecords/locationHistory.destination.json

```diff
@@ -164,166 +164,146 @@
+      "ca_movements.MovementDate":"20 October 1999",
       "ca_movements.LocationStatus":"Displayed",
-      "ca_objects":"A1949.11o",
+      "ca_objects":"A1984.380",
       "ca_objects.type_id":"Artwork",
-      "locationHierarchy":"Hall, North Wall",
-      "ca_storage_locations":"Hall, North Wall",
-      "ca_storage_locations.type_id":"container"
+      "locationHierarchy":"Foyer",
+      "ca_storage_locations":"Foyer",
+      "ca_storage_locations.type_id":"room"
     },
     {
       "movement_id":"14",
       "id":"14",
-      "idno":"121",
+      "idno":"170",
       "display_label":"???",
       "ca_movements.type_id":"Movement",
-      "ca_movements.idno":"121",
-      "ca_movements.AuthorizedBy":"Curator",
-      "ca_movements.MovementDate":"5 November 1998",
-      "ca_movements.LocationStatus":"Stored",
-      "ca_objects":"A1949.11o",
-      "ca_objects.type_id":"Artwork",
-      "locationHierarchy":"Locked Storeroom;B1",
-      "ca_storage_locations":"B1",
-      "ca_storage_locations.type_id":"container"
+      "ca_movements.idno":"170",
+      "ca_movements.AuthorizedBy":"CURATOR",
+      "ca_movements.MovementDate":"18 April 2000",
+      "ca_movements.LocationStatus":"Displayed",
+      "locationHierarchy":"Tranby room",
+      "ca_storage_locations":"Tranby room",
+      "ca_storage_locations.type_id":"room"
     },
     {
       "movement_id":"15",
       "id":"15",
-      "idno":"155",
+      "idno":"1475",
       "display_label":"???",
       "ca_movements.type_id":"Movement",
-      "ca_movements.idno":"155",
-      "ca_movements.AuthorizedBy":"Curator",
-      "ca_movements.MovementDate":"20 October 1999",
-      "ca_movements.LocationStatus":"Displayed",
-      "ca_objects":"A1978.9",
-      "ca_objects.type_id":"Artwork",
-      "locationHierarchy":"Top room",
-      "ca_storage_locations":"Top room",
-      "ca_storage_locations.type_id":"container"
+      "ca_movements.idno":"1475",
+      "ca_movements.AuthorizedBy":"AndI",
+      "ca_movements.MovementDate":"3 May 2002",
+      "ca_movements.LocationStatus":"Stored",
+      "locationHierarchy":"Imp 1",
+      "ca_storage_locations":"Imp 1",
+      "ca_storage_locations.type_id":"shelf"
     },
     {
       "movement_id":"16",
       "id":"16",
-      "idno":"156",
+      "idno":"3535",
       "display_label":"???",
       "ca_movements.type_id":"Movement",
-      "ca_movements.idno":"156",
-      "ca_movements.AuthorizedBy":"Curator",
-      "ca_movements.MovementDate":"20 October 1999",
+      "ca_movements.idno":"3535",
+      "ca_movements.AuthorizedBy":"YokK",
+      "ca_movements.MovementDate":"8 January 2004",
       "ca_movements.LocationStatus":"Stored",
-      "ca_objects":"A1978.9",
-      "ca_objects.type_id":"Artwork",
-      "locationHierarchy":"Locked Storeroom;Drawer B3",
-      "ca_storage_locations":"Drawer B3",
-      "ca_storage_locations.type_id":"drawer"
+      "ca_objects":"MA1984.331ab",
+      "ca_objects.type_id":"Museum Artefact",
```

Comment:
Here's a museum artefact with location history. Evidence that `RWAHS-141` is fixed.